src/zenml/utils/source_utils.py

```diff
@@ -308,13 +308,9 @@ def is_distribution_package_file(file_path: str, module_name: str) -> bool:
         if Path(path).resolve() in absolute_file_path.parents:
             return True
 
-    if _get_package_for_module(module_name=module_name):
-        return True
-
-    # TODO: Both of the previous checks don't detect editable installs because
+    # TODO: The previous checks does not detect editable installs because
```

Comment:
"check does" or "check do" 😛 

```suggestion
    # TODO: The previous check does not detect editable installs because
```